### PR TITLE
[VecOps] Introduce Range(begin, end) helper function

### PR DESCRIPTION
**Pull request by eguiraud**

### CODE REVIEW

File: math/vecops/inc/ROOT/RVec.hxx [hunk 1]
```diff
@@ -3043,6 +3043,17 @@ inline RVec<std::size_t> Range(std::size_t length)
    return ret;
 }
 
+/// Produce RVec with entries equal to begin, begin+1, ..., end-1.
+/// An empty RVec is returned if end >= begin.
```
Comment (vepadulano) [inline math/vecops/inc/ROOT/RVec.hxx:3047]:
```suggestion
/// An empty RVec is returned if begin >= end.
```

**Review by vepadulano — APPROVED** [review]
LGTM, see minor comment

**Review by ikabadzhov — APPROVED** [review]
lgtm considering Vincenzo's comment

File: math/vecops/inc/ROOT/RVec.hxx (hunk 1 above)
Comment (eguiraud) [reply math/vecops/inc/ROOT/RVec.hxx:3047]: thanks!

**Pull request merged**